packages/tasks/src/tasks/text-to-speech/about.md

```diff
@@ -62,4 +62,5 @@ await inference.textToSpeech({
 - [A guide on Fine-tuning Whisper For Multilingual ASR with 🤗Transformers](https://huggingface.co/blog/fine-tune-whisper)
 - [Speech Synthesis, Recognition, and More With SpeechT5](https://huggingface.co/blog/speecht5)
 - [Optimizing a Text-To-Speech model using 🤗 Transformers](https://huggingface.co/blog/optimizing-bark)
+- [Train your own TTS models with Parler-TTS](https://github.com/huggingface/parler-tts)
```

Comment:
Should we reorganize the useful resources section? 
There's the guide on fine-tuning whisper that should be put last since it's not directly related to TTS!

Reply:
Done! Good shout!